open-reac/src/test/java/com/powsybl/openreac/optimization/OpenReacOptimizationIndicatorsTest.java

```diff
@@ -0,0 +1,280 @@
+        assertEquals(3, Integer.parseInt(result.getIndicators().get("nb_unit_in_data_file")));
+        assertEquals(3, Integer.parseInt(result.getIndicators().get("nb_unit_in_AC_CC")));
+        assertEquals(3, Integer.parseInt(result.getIndicators().get("nb_unit_up_and_running")));
+        // verify that only the generators indicated as constant and with defined target Q are fixed in optimization
+        assertEquals(1, Integer.parseInt(result.getIndicators().get("nb_unit_with_fixed_reactive_power")));
+        // verify other generators are optimized
+        assertEquals(2, Integer.parseInt(result.getIndicators().get("nb_unit_with_variable_reactive_power")));
+    }
+
+    @Test
+    void testTransformersIndicators() throws IOException {
+        Network network = VoltageControlNetworkFactory.createNetworkWith2T2wt();
+        OpenReacParameters parameters = new OpenReacParameters();
+        parameters.addVariableTwoWindingsTransformers(List.of("T2wT1"));
+        OpenReacResult result = runOpenReac(network, "optimization/indicators/transfo-test", parameters, true);
+
+        // verify only one rtc has been optimized
+        assertEquals(OpenReacStatus.OK, result.getStatus());
+        assertEquals(1, Integer.parseInt(result.getIndicators().get("nb_transformers_with_variable_ratio")));
+        assertEquals(1, Integer.parseInt(result.getIndicators().get("nb_transformers_with_fixed_ratio")));
+    }
+
+    @Test
+    void testVscIndicators() throws IOException {
+        Network network = HvdcNetworkFactory.createVsc();
+        // FIXME : should not depend on P/Q values on terminal
+        network.getVscConverterStation("cs2").getTerminal().setP(2).setQ(1);
+        OpenReacResult result = runOpenReac(network, "optimization/indicators/vsc-test", true);
+
+        assertEquals(OpenReacStatus.OK, result.getStatus());
+        assertEquals(2, Integer.parseInt(result.getIndicators().get("nb_vsc_converter_in_data_file")));
+        // verify only vsc in main cc is optimized
+        assertEquals(1, Integer.parseInt(result.getIndicators().get("nb_vsc_converter_up_and_running")));
+    }
+
+    @Test
+    void testLccIndicators() throws IOException {
+        Network network = HvdcNetworkFactory.createLcc();
+        // FIXME : should not depend on P/Q values on terminal
+        network.getLccConverterStation("cs2").getTerminal().setP(2).setQ(1);
+        OpenReacResult result = runOpenReac(network, "optimization/indicators/lcc-test", true);
+
+        assertEquals(OpenReacStatus.OK, result.getStatus());
+        assertEquals(2, Integer.parseInt(result.getIndicators().get("nb_lcc_converter_in_data_file")));
+        // verify only lcc in main cc is considered in optimization
+        assertEquals(1, Integer.parseInt(result.getIndicators().get("nb_lcc_converter_up_and_running")));
+    }
+
+    @Test
+    void testShuntsIndicators() throws IOException {
+        Network network = OpenReacOptimizationAndLoadFlowTest.create();
```

Comment:
Ok I think I get why this function was defined as public... Maybe it should be somewhere else then to be reachable from both test classes?

Reply:
Solved [here](https://github.com/powsybl/powsybl-optimizer/pull/83#discussion_r1899475643).